### PR TITLE
Add conversation:start webhook trigger

### PR DESCRIPTION
**Pull request by lemieux**
Hopefully it doesn't clash too much with #228.

### CODE REVIEW

File: slate/source/includes/rest/_core.md [hunk 1]
```diff
@@ -417,6 +417,7 @@ A webhook will make a request to the target each time a trigger associated with
 | **message**<br/>*default* | all messages                                                   |
 | **message:appUser**       | only messages with role `appUser`                              |
 | **message:appMaker**      | only messages with role `appMaker` or `whisper`                |
+| **conversation:start**    | when a user opts in to start receiving messages                |
```
Comment (jpjoyal) [inline slate/source/includes/rest/_core.md]: missing note about channel availability

Comment (lemieux) [inline slate/source/includes/rest/_core.md]: what note? there's no note for conversation:read.. should it just go in the grid?

File: slate/source/includes/rest/_core.md [hunk 2]
```diff
@@ -740,6 +741,40 @@ The payload for when a user sends their location.
...
+            "lastSeen": "2016-03-09T19:09:01.431Z",
+            "platform": "messenger"
+          }
+        ]
+    },
+    "timestamp": 1480349392.103
+}
+```
+
+The payload for when a user opts in to start receiving messages.
```
Comment (jpjoyal) [inline slate/source/includes/rest/_core.md]: The note that details availability (and unavailability on SDKs can go here)

Comment (lemieux) [inline slate/source/includes/rest/_core.md]: Just a note in text or a big `aside` info bubble?

Comment (jpjoyal) [inline slate/source/includes/rest/_core.md]: Thinking more about it, I like the idea of adding this to the grid. Under indicators, add Conversation start which links back to this webhook payload. The note can just normal text along the line : "Conversation start is only available on a sub set of channels. [Channel capabilities](https://docs.smooch.io/guide/channel-capabilities/) lists which channel currently supports it."

File: slate/source/includes/rest/_core.md [hunk 3]
```diff
@@ -740,6 +741,40 @@ The payload for when a user sends their location.
...
+            "lastSeen": "2016-03-09T19:09:01.431Z",
+            "platform": "messenger"
+          }
+        ]
+    },
+    "timestamp": 1480349392.103
+}
+```
+
+The payload for when a user opts in to start receiving messages. `conversation:start` is only available on a sub set of channels. [Channel capabilities](https://docs.smooch.io/guide/channel-capabilities/) lists which channel currently supports it.
```
Comment (jpjoyal) [inline slate/source/includes/rest/_core.md]: worth adding a note about the fact that the event is not emitted when a channel is being linked

File: slate/source/includes/rest/_core.md [hunk 4]
```diff
@@ -740,6 +741,40 @@ The payload for when a user sends their location.
...
+            "lastSeen": "2016-03-09T19:09:01.431Z",
+            "platform": "messenger"
+          }
+        ]
+    },
+    "timestamp": 1480349392.103
+}
+```
+
+The payload for when a user opts in to start receiving messages. `conversation:start` is only available on a sub set of channels. [Channel capabilities](https://docs.smooch.io/guide/channel-capabilities/) lists which channel currently supports it. Also, note that `conversation:start` won't be triggered during the channel linking flow.
```
Comment (jpjoyal) [inline slate/source/includes/rest/_core.md]: suggest "during the channel linking flow." -> "when a user is linking a second channel via Web Messenger" to be more specific. All good after that

Comment (lemieux) [inline slate/source/includes/rest/_core.md]: done

**Pull request merged**